app/views/accounts/_header.html.erb

```diff
@@ -1,11 +1,11 @@
-<div class="grid-row padding-top-2 padding-bottom-4 tablet:padding-y-0 margin-bottom-4 bg-lightest-blue sm-bg-none position-relative">
+<div class="grid-row padding-y-2 tablet:padding-y-0 margin-bottom-4 bg-lightest-blue sm-bg-none position-relative">
   <div class="grid-col-12 tablet:grid-col-fill">
     <h1 class="margin-0 center sm-left-align">
       <span class="regular tablet:display-none">
         <%= t('account.welcome') %>,
-        <span class="bold">
+        <b class="text-wrap-anywhere">
```

Comment:
I thought we were moving more towards `<strong>` than `<b>`?

Reply:
> I thought we were moving more towards `\<strong\>` than `<b>`?

Typically, yes, `<b>` is smelly and usually better implemented as `<strong>` in the case that it's something of particular importance. I sorta felt this met the limited use-case of a `<b>` tag though, which is something which is cosmetically bold, but not of any real importance.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b

Though in retrospect, maybe it would be better to implement as `<span class="text-bold">` to avoid any future developers facing this dilemma or, worse, copying it to use in a place where `<strong>` would be more suitable.

The main goal was to remove the BassCSS class 🤷

Reply:
> https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b

On a re-read, maybe using it this way isn't encouraged, at least if our aim is bolded text. That being said, the article seems self-contradictory in some places.

>you should not use `<b>` for styling text; instead, you should use the CSS [font-weight](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight) property to create boldface text

>Use the `<b>` for cases like [...] spans of text whose typical presentation would be boldfaced

Reply:
`<span>` for the win